packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/internal/KtorNetworkTransport.kt

```diff
@@ -66,6 +67,14 @@ class KtorNetworkTransport(
         usesRefreshToken: Boolean
     ): Response {
         try {
+            // FIXME When using a shared HttpClient we are seeing sporadic
```

Comment:
Is this a placeholder for investigations as part of this PR? If not, then I think that it at least should have its own issue to track it. 

Maybe also only add the work around for `macos` if that is the only place we are seeing it.

Reply:
Yes, I was thinking about making some sort of platform-specific strategy here as well. Will do that. I'll also create an issue for investigating the root cause.